ISHPermissionKit/Requests/ISHPermissionRequestAccount.m

```diff
@@ -38,6 +48,7 @@ - (NSString *)accountTypeIdentifier {
             return ACAccountTypeIdentifierTencentWeibo;
             
         default:
+            NSAssert(NO, @"Invalid category: %@", @(self.permissionCategory));
```

Comment:
The code above probably raises a warning... we don't really want to list all ~15 categories, so I believe `default` is ok here. Plus, we usually don't care here if new categories are added.